src/chains/opStack/errors/withdrawal.ts

```diff
@@ -0,0 +1,15 @@
+import { BaseError } from '../../../errors/base.js'
+import type { Hex } from '../../../types/misc.js'
+
+export type ReceiptContainsNoWithdrawalsErrorType =
+  ReceiptContainsNoWithdrawalsError & {
+    name: 'ReceiptContainsNoWithdrawalsError'
+  }
```

Comment:
```suggestion
  ReceiptContainsNoWithdrawalsError
```

Why is the & necessary?

Reply:
To discriminate the type.